src/components/vm/hostdevs/hostDevAdd.jsx

```diff
@@ -110,8 +110,23 @@ const DevRow = ({ idPrefix, type, selectableDevices, setSelectableDevices }) =>
                    aria-label="Selectable Table"
                    cells={['', '', '', '']}
                    rows={selectableDevices.map((dev, idx) => {
-                       return { selected: dev.selected, cells: [dev.nodeDev.capability.product._value, dev.nodeDev.capability.vendor._value, ...getSource(dev.nodeDev, idx)] };
-                   })}>
+                       // Filtered matchers should be USB hub, PCI bridge;
+                       // FIXME: both should filtered by class, not string...
+                       // but USB isn't exposing class at the moment (only PCI does?)
+                       if (dev.nodeDev.capability.product._value &&
+                        !dev.nodeDev.capability.product._value.match(/^hub$| hub$|^$/i) &&
+                        !(dev.nodeDev.class || "").match(/bridge$/i)) {
+                           console.info(dev.nodeDev.class || "UNDEFINED");
+                           console.table(dev.nodeDev);
```

Comment:
Obviously, we should remove the console calls before merging. I think it's still useful for debugging during development right now.